Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageSendButton_Tests.swift

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import SwiftUI
+import XCTest
+
+class ChatMessageSendButton_Tests: XCTestCase {
+    private lazy var container = UIView().withoutAutoresizingMaskConstraints
+
+    override func setUp() {
+        super.setUp()
+        container.subviews.forEach { $0.removeFromSuperview() }
+    }
+
+    func test_defaultAppearance_newMessage() {
+        let view = ChatMessageSendButton().withoutAutoresizingMaskConstraints
+        container.embed(view)
+
+        view.isEnabled = true
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-enabled")
+
+        view.isEnabled = false
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-disabled")
+    }
+
+    func test_appearanceCustomization_usingUIConfig() {
+        var config = UIConfig()
+        config.images.messageComposerSendMessage = UIImage(
+            named: "reaction_love_big",
+            in: .streamChatUI
+        )!
+            .tinted(with: .systemPink)!
+        config.colorPalette.inactiveTint = .black
+
+        let view = ChatMessageSendButton().withoutAutoresizingMaskConstraints
+        view.uiConfig = config
+
+        container.embed(view)
+
+        view.isEnabled = true
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-enabled")
+
+        view.isEnabled = false
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-disabled")
+    }
+
+    func test_appearanceCustomization_usingAppearanceHook() {
+        class TestView: ChatMessageSendButton {}
+        TestView.defaultAppearance {
+            $0.tintColor = .green
+        }
+
+        let view = TestView().withoutAutoresizingMaskConstraints
+
+        container.embed(view)
+
+        view.isEnabled = true
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-enabled")
+
+        view.isEnabled = false
+        AssertSnapshot(container, variants: .onlyUserInterfaceStyles, suffix: "new-disabled")
+    }
+
+    func test_customizationUsingSubclassingHook() {
```

Comment:
Probably better to rename it to `test_appearanceCustomization_usingSubclassing` for consistency with the other audits.